lib/ddsl/bundle.js

```diff
@@ -140,7 +140,7 @@ BEMXJST.prototype.runSimple = function runSimple(context) {
 DDSL.prototype.render = function render(
   context, entity, tag, js, bem, cls, mix, attrs, content
 ) {
-
+console.log(attrs);
```

Comment:
It is a bundle:) don't worry

Sent from my iPhone

> On 14 Jul 2016, at 00:40, Vasiliy <notifications@github.com> wrote:
> 
> In lib/ddsl/bundle.js:
> 
> > @@ -140,7 +140,7 @@ BEMXJST.prototype.runSimple = function runSimple(context) {
> >  DDSL.prototype.render = function render(
> >    context, entity, tag, js, bem, cls, mix, attrs, content
> >  ) {
> > -
> > +console.log(attrs);
> ?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>